src/SW_Weather.c

```diff
@@ -781,6 +867,108 @@ void generateMissingWeather(
   }
 }
 
+/**
+ @brief Check weather through all years/days within simultation and make sure all input values are reasonable
+ after possible weather generation and scaling. If a value is to be found unreasonable, the function will execute a program crash.
+
+ @param weather Struct of type SW_WEATHER holding all relevant information pretaining to weather input data
+ */
+void checkAllWeather(SW_WEATHER *weather) {

```

Comment:
This needs to be able to ignore missing weather values (at least for max/min temperature and ppt, and probably for all).